### PR TITLE
CER-2116 Fix html report not created on successful first run.

### PR DESCRIPTION
**Pull request by domiebett**
Resolves [CER-2116](https://cerosteam.atlassian.net/browse/CER-2116)

This works alongside https://github.com/ceros/ceros-product/pull/6499 as the options to get this working are set in the baseConf.js.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -369,6 +369,18 @@ function Jasmine2HTMLReporter(options) {
         exportObject.endTime = new Date();
     };
 
+    self.afterLaunch = function(callback) {
+        // write success report. if all test pass and showFailuresOnly options is true,
+        // file wont be created.
```
Comment (addelong) [inline index.js]: I'm confused, it appears that on line 377, if showFailuresOnly is true, the file WILL be created.

File: static/success.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="container">
+    <div class="checkmark-holder">
+      <div class="checkmark-circle"></div>
+      <div class="checkmark-stem"></div>
+      <div class="checkmark-kick"></div>
+    </div>
+    <div class="message-holder"><span class="message">Congratulations, All your tests passed...!!!</span></div>
```
Comment (addelong) [inline static/success.html]: lol the ellipsis makes it sound ominous, can you change it to just something like `All tests passed!`

**Review by addelong — COMMENTED** [review]
back to you @domiebett

File: static/success.html (hunk 2 above)
Comment (domiebett) [reply static/success.html]: 😄 ... cool. Let me do that.

File: index.js (hunk 1 above)
Comment (domiebett) [reply index.js]: Yeah... its supposed to mean that currently the file is not created if both options are true. that is a confusing comment. Let me update that. Thanks.

**Comment by domiebett** [comment]
@addelong , I have made the changes.

**Comment by addelong** [comment]
Code 👍

**Pull request merged**